### PR TITLE
Change DogeAPI to Block.io

### PR DESCRIPTION
**Pull request by Kirvx**
I added Block.io instead of DogeAPI in Getting Started (images, description + french description).
I'm not a web developper but i think it's ok....
Maybe you can correct few things and correct my english translations, since I'm french...

I fixed the problem https://github.com/dogecoin/dogecoin.com/pull/52#issuecomment-58221235

### CODE REVIEW

**Comment by PowerLemons** [comment]
Your English translation is a bit off. I'll create a correction for your pull request with a pull request when I have the time to do so, but everything else seems okay.

**Comment by Kirvx** [comment]
I'm really not comfortable with Github, so please merge and corrects https://github.com/Kirvx/dogecoin.com/commit/e20e02701e240fe3720d64bc299c5c0e93daaaec ASAP with the remove of Moolah https://github.com/dogecoin/dogecoin.com/commit/4c4e721d5af65ea3f6da17b7aeea86eeb865c02b and english fixes before i make a stupidity.

**Comment by Kirvx** [comment]
I will open a new pull request, please tell me what i have to change for english sentences.

File: getting-started/index.html [hunk 1]
```diff
@@ -576,22 +579,35 @@ <h5 data-i18n="dgc-footer-copyright">© The Ðogecoin Project</h5>
 					<img style="padding-bottom: 30px;" src="../imgs/tut/dogewallet_signup_02.jpg" class="img-responsive center-block" alt="Dogecoin Wallet Launch" />
 				</div>
 				<div class="col-md-8">
-					<h2><strong data-i18n="dgc-getting-started-web-blockio-2-0">2. Verify Your Account.</strong></h2>
-					<p data-i18n="dgc-getting-started-web-blockio-2-1">After registering, you will be asked to confirm your account with a verification email.</p>
-					<p data-i18n="dgc-getting-started-web-blockio-2-2">Go to your email account and click the verification link in the email.</p>
+					<h2><strong data-i18n="dgc-getting-started-web-blockio-2-0">2. Set Your Secret PIN.</strong></h2>
+					<p data-i18n="dgc-getting-started-web-blockio-2-1">Set your secret PIN with 8 characters at least, it must contain only numbers and letters.</p>
+					<p data-i18n="dgc-getting-started-web-blockio-2-2">Then, click on Set Secret PIN.</p>
```
Comment (dhpollack) [inline getting-started/index.html:584]:
```
"Set your secret PIN with 8 characters at least, it must contain only numbers and letters."
Create a Secret PIN containing at least 8 characters (only numbers and letters are allowed).

"Then, click on Set Secret PIN."
Then click Set Secret PIN
```

File: getting-started/index.html [hunk 2]
```diff
@@ -576,22 +579,35 @@ <h5 data-i18n="dgc-footer-copyright">© The Ðogecoin Project</h5>
...
 		<hr>
+        <div class="row">
+			<div style="padding-top: 30px;">
+				<div class="col-md-4">
+					<img style="padding-bottom: 30px;" src="../imgs/tut/dogewallet_signup_03.jpg" class="img-responsive center-block" alt="Dogecoin Wallet Launch" />
+				</div>
+				<div class="col-md-8">
+					<h2><strong data-i18n="dgc-getting-started-web-blockio-3-0">3. Note your Secret Mnemonic.</strong></h2>
+					<p data-i18n="dgc-getting-started-web-blockio-3-1">Keep a safe location of the Secret Mnemonic that was generated.</p>
+					<p data-i18n="dgc-getting-started-web-blockio-3-2">Click the checkbox to know that if you forget your PIN or Mnemonic code, you will lose your coins.</br> Click the Done button.</p>
```
Comment (dhpollack) [inline getting-started/index.html:597]:
```
"Keep a safe location of the Secret Mnemonic that was generated."
Keep the generated Secret Mnemonic in a safe location.

"Click the checkbox to know that if you forget your PIN or Mnemonic code, you will lose your coins.</br> Click the Done button."
Check the box to confirm that you are aware that if you lose your PIN and Mnemonic, then you will lose access to your coins.  <br />Click the done button.
```

**Pull request closed without merging**